src/typing/calls.ml

```diff
@@ -28,6 +28,16 @@ let make_call ctx e params t ?(force_inline=false) p =
 			| _ ->
 				raise Exit
 		in
+		(match cl, ctx.curclass.cl_kind, params with
```

Comment:
This code is misplaced here because we don't even know if it's an inline function at this point. Furthermore, I don't think we necessarily have processed the field's expression at this point, so the class field flag may not be set yet.

This should be moved below the `ignore(follow f.cf_type);` line further below because the `follow` guarantees that the expression is evaluated.

Reply:
Done